### PR TITLE
Upgrade lock

### PR DESCRIPTION
**Pull request by 4z0t**
Fixes quick dialog locking screen when removing upgrade of a unit being destroyed.

### CODE REVIEW

File: lua/ui/game/construction.lua [hunk 1]
```diff
@@ -1184,7 +1198,8 @@ function watchForQueueChange(unit)
         end
 
         local selection = GetSelectedUnits() or {}
-        if lastDisplayType and table.getn(selection) == 1 and threadWatchingUnit == watchingUnit and selection[1] == threadWatchingUnit then
+        if lastDisplayType and table.getn(selection) == 1 and threadWatchingUnit == watchingUnit and
```
Comment (Garanas) [inline lua/ui/game/construction.lua:1201]: what happened here?

**Comment by Garanas** [comment]
What happened to the original formatting?

**Comment by 4z0t** [comment]
> What happened to the original formatting?

I use foramtting on save xd

**Comment by Garanas** [comment]
Could you make a pull request with just the changes involved?

**Comment by 4z0t** [comment]
I will try to force push

**Comment by Garanas** [comment]
@4z0t any time - I'd like to include this for the release next weekend.

**Pull request merged**